samples/wifi/raw_tx_packet/src/main.c

```diff
@@ -125,6 +125,27 @@ static void wifi_set_mode(int mode_val)
 	}
 }
 
+#ifdef CONFIG_RAW_TX_PKT_SAMPLE_INJECTION_ENABLE
+static int wifi_set_tx_injection_mode(void)
+{
+	struct net_if *iface;
+
+	iface = net_if_get_first_wifi();
+	if (!iface) {
+		LOG_ERR("Failed to get Wi-Fi iface");
+		return -1;
+	}
+
+	if (net_eth_txinjection_mode(iface, true)) {
```

Comment:
commit logs says het_shell, but its net API, there isn't a shell command for this.